### PR TITLE
Update service questions for DOS 5

### PR DESCRIPTION
**Pull request by bjgill**
Trello: https://trello.com/c/yWkgqyPL/325-2-update-dos5-application-content-in-the-frameworks-repo

We need to make a number of changes following the accessibility review and CCS requests. See the spreadsheet linked from the Trello card for details.

This PR now covers all the service question changes requested, except the reports of error oddities. I've not been able to reproduce them. Let's get this into Preview and see if they show up there.

Don't try and review this all together. I'd recommend going through commit by commit instead - they're all independent of each other.

### CODE REVIEW

**Comment by lfdebrux** [comment]
### My review checklist

I've:

- [ ] looked at the changes
- [ ] tested locally

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "digitalmarketplace-frameworks",
-  "version": "17.15.0",
+  "version": "17.16.0",
```
Comment (lfdebrux) [inline package.json:3]: Can we split the version bump into a separate commit please?

File: frameworks/digital-outcomes-and-specialists-5/questions/services/accessibleApplicationsOutcomes.yml [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
-  Read guidance about how to meet the regulations and <a target="_blank" rel="noopener noreferrer" href="https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps">design and build accessible services</a>.
+  Read guidance about how to meet the regulations and <a target="_blank" rel="noopener noreferrer" href="https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps">design and build accessible services (opens new tab)</a>.
 type: boolean
 required_value: true
 
 depends:
   - "on": "lot"
     being:
       - digital-outcomes
+      - digital-specialists
```
[the remark this answers is not in the document]
Comment (bjgill) [reply frameworks/digital-outcomes-and-specialists-5/questions/services/accessibleApplicationsOutcomes.yml:13]: We were asked to make this change: https://docs.google.com/spreadsheets/d/1FNGwe56myW2ddO6XQUtex209TJQ2Ub99PTzcWxpejsQ/edit#gid=866429329&range=D55

File: frameworks/digital-outcomes-and-specialists-5/manifests/edit_service.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 -
-  name: Lab name
+  name: Research studio name
```
[the remark this answers is not in the document]
Comment (bjgill) [reply frameworks/digital-outcomes-and-specialists-5/manifests/edit_service.yml:2]: I checked with @NoraGDS about this, who agreed that this was a good idea.

[the remark this answers is not in the document]
Comment (bjgill) [reply frameworks/digital-outcomes-and-specialists-5/manifests/edit_service.yml:2]: Hmm. Actually, looking at the introduction to the lab section:

> User research studios sometimes have multiple labs, or rooms. Labs usually include a device to test a service and a way to observe the session. Provide information on each lab you want buyers to use.

On that basis, it seems as if 'research studio' is the building in which the 'lab' is located. Unless we want to change the definition of 'lab', we should probably keep everything as-is

[the remark this answers is not in the document]
Comment (bjgill) [reply frameworks/digital-outcomes-and-specialists-5/manifests/edit_service.yml]: I discussed this with @NoraGDS. The key bit of background information is that CCS want everything to be 'research studios'. 

To do this, we'll need to re-write a couple of bits of content so they still make sense. So I'm going to leave this PR using 'lab' and handle the renaming in a follow-on PR.

File: frameworks/digital-outcomes-and-specialists-5/questions/services/accessibleApplicationsOutcomes.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (lfdebrux) [reply frameworks/digital-outcomes-and-specialists-5/questions/services/accessibleApplicationsOutcomes.yml:13]: Can we remove the duplicate question for each specialists to avoid asking the same question twice?

```
$ find questions/services -name '*AccessibleApplications.yml'
questions/services/contentDesignerAccessibleApplications.yml
questions/services/webOperationsAccessibleApplications.yml
questions/services/technicalArchitectAccessibleApplications.yml
questions/services/designerAccessibleApplications.yml
questions/services/qualityAssuranceAccessibleApplications.yml
questions/services/developerAccessibleApplications.yml
questions/services/serviceManagerAccessibleApplications.yml
```

**Comment by bjgill** [comment]
@lfdebrux - thanks for the review. I think I've dealt with everything you noticed. Does this look better now?

I'm hoping to get this merged and released by the end of the day.

File: frameworks/digital-outcomes-and-specialists-5/questions/services/accessibleApplicationsOutcomes.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (bjgill) [reply frameworks/digital-outcomes-and-specialists-5/questions/services/accessibleApplicationsOutcomes.yml:13]: Ah - good catch. I hadn't noticed that some of the specialists had their own questions. Removed.

**Review by lfdebrux — APPROVED** [review]
Alright, let's get it on preview for proper testing
